packages/blocks/tests/php/test-blocks.php

```diff
@@ -84,4 +111,116 @@ public function test_is_amp_request() {
 	public function test_is_not_amp_request() {
 		$this->assertFalse( Blocks::is_amp_request() );
 	}
+
+	/**
+	 * Test WordPress and Gutenberg version requirements.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::is_gutenberg_version_available
+	 */
+	public function test_returns_false_if_core_wp_version_less_than_minimum() {
+		$version_gated = Blocks::is_gutenberg_version_available(
+			array(
+				'wp'        => '999999',
+				'gutenberg' => '999999',
+			),
+			'gated_block'
+		);
+		$this->assertEquals( false, $version_gated );
+	}
+
+	/**
+	 * Test WordPress and Gutenberg version requirements.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::is_gutenberg_version_available
+	 */
+	public function test_returns_true_if_core_wp_version_greater_or_equal_to_minimum() {
+		$version_gated = Blocks::is_gutenberg_version_available(
+			array(
+				'wp'        => '1',
+				'gutenberg' => '999999',
+			),
+			'ungated_block'
+		);
+		$this->assertEquals( true, $version_gated );
+	}
+
+	/**
+	 * Testing removing the Jetpack prefix from a block slug.
+	 *
+	 * @covers Automattic\Jetpack\Blocks::remove_extension_prefix
+	 *
+	 * @dataProvider get_extension_name_provider
+	 *
+	 * @param string $extension_slug      Block / Extension name.
+	 * @param string $expected_short_slug Extension name without Jetpack prefix.
+	 */
+	public function test_remove_extension_prefix( $extension_slug, $expected_short_slug ) {
+		$short_slug = Blocks::remove_extension_prefix( $extension_slug );
+
+		$this->assertEquals( $short_slug, $expected_short_slug );
```

Comment:
PHPUnit's `assertEquals()` takes the "expected" value first, and the value being tested second. If you switch them around, the failure message comes out wrong.

```suggestion
		$this->assertEquals( $expected_short_slug, $short_slug );
```

Reply:
Somehow I always mix those up, thank you!